main/templates/main/snippets/footer.html

```diff
@@ -96,7 +96,7 @@ <h4 class="h6 fw-bold pb-2 mb-0 mb-lg-1">Install App</h4>
             </div>
         </div>
         <p class="nav fs-sm mb-0">
-            <span class="text-body-secondary">&copy; Digital Research Competencies Framework 2023 - {% now "Y" %}</span>
+            <span class="text-body-secondary">© Digital Research Competencies Framework 2023 - {% now "Y" %}</span>
```

Comment:
I'm not sure about this character. Any more html-savvy people know what the best choice is here?

Reply:
I have historically tended to use the encoding, but it is actually recommended by djlint to not use entity references ([H023](https://www.djlint.com/docs/linter/)). So let's stick with that to avoid problems.